src/shared/components/data-table/components/main/components/table-header/components/table-header-cell/TableHeaderCell.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import classNames from 'classnames';
+import React, { useContext } from 'react';
+
+import QuestionButton from 'src/shared/components/question-button/QuestionButton';
+import {
+  type IndividualColumn,
+  SortingDirection
+} from 'src/shared/components/data-table/dataTableTypes';
+import { TableContext } from 'src/shared/components/data-table/DataTable';
+
+import SortIcon from 'static/icons/icon_arrow.svg';
+
+import styles from '../../TableHeader.scss';
+
+const TableHeaderCell = (props: IndividualColumn) => {
+  const { title, helpText, isSortable, columnId, width } = props;
+
+  const { dispatch, sortedColumn } = useContext(TableContext) || {};
+
+  const { columnId: sortedColumnId, sortedDirection } = sortedColumn || {};
+
+  if (!dispatch) {
+    return null;
+  }
+
+  let currentColumnSortingDirection = SortingDirection.DEFAULT;
+  if (
+    columnId === sortedColumnId &&
+    sortedDirection &&
```

Comment:
`sortedDirection` here seems redundant here since the next line is checking if it is `SortingDirection.ASC` or not

Reply:
Removed it